lua/nvim-possession/utils.lua

```diff
@@ -104,4 +108,17 @@ M.autoswitch = function(config)
 	end
 end
 
+---converts a session name into a session object, containing the name and
+---last modified time.
+---@param sessions_path string
+---@param name string
+---@return table | nil
+M.name_to_session = function(sessions_path, name)
```

Comment:
Do we need an extra method here? Currently if you have multiple sessions in the same working directory they are indeed shown by the picker already. The comment to the function says

> converts a session name into a session object

the session object is however already created by default, isn't it?

Reply:
It's mostly just a helper to reduce some code duplication -- in `M.list()` as well as `util.sessions_in_cwd`, there is code that fetches just the filename of the session file, and then the file is statted to get its modification time. Since there were two locations where we need to convert a filename to an object containing the name and its modification time, I moved it into a helper.

Maybe the term `session_object` isn't the most clear --- it's just the object we need to display the session name but also be able to sort by modified time.